funding/batch.go

```diff
@@ -48,13 +61,20 @@ type batchChannel struct {
 func (c *batchChannel) processPsbtUpdate(u *lnrpc.OpenStatusUpdate) error {
 	psbtUpdate := u.GetPsbtFund()
 	if psbtUpdate == nil {
-		return fmt.Errorf("got unexpected channel update %v", u.Update)
+		return &errorBatchChannel{
+			peer: c.fundingReq.TargetPubkey.SerializeCompressed(),
```

Comment:
drive-by comment, are we confident that this always doesn't result in a nil-pointer?